test/end-to-end/company.js

```diff
@@ -450,6 +466,51 @@ describe("Company endpoint", () => {
                 expect(res.body).toHaveProperty("offers");
                 expect(res.body.offers.length).toEqual(0);
             });
+
+            test("should return if a disabled company returns 403", async () => {
+                await test_agent
+                    .post("/auth/login")
+                    .send(test_user)
+                    .expect(HTTPStatus.OK);
+
+                await Promise.all(Array(CompanyConstants.offers.max_profile_visible - 1).fill(null).map((_) =>
+                    Offer.create(
+                        generateTestOffer({
+                            "publishDate": (new Date(Date.now() - (DAY_TO_MS))).toISOString(),
+                            "publishEndDate": (new Date(Date.now() + (DAY_TO_MS))).toISOString(),
+                            owner: disabled_company._id,
+                            ownerName: disabled_company.name,
+                            ownerLogo: disabled_company.logo
+                        })
+                    )).sort((a1, a2) => a1.publishDate > a2.publishDate));
+
+                await test_agent
```

Comment:
You should get the response object given by this method chain in order to perform further assertion: comparing the real error message with the expected one, etc...